docs/source/quick-start.mdx

```diff
@@ -0,0 +1,96 @@
+# Quick start
+
+The Hugging Face Hub is the go-to place for sharing machine learning models, datasets, and metrics. To help you get the most out of the Hub, we created the `huggingface_hub` library so you can interact with the Hub without leaving your development environment. You can create and manage repositories easily and download and upload files from the Hub. 
+
+## Install the Hub library
+
+To get started, you should install the `huggingface_hub` library. The library provides functions that make interacting with the Hub simple. Run the following command to install the library:
+
+```bash
+>>> pip install huggingface_hub
+```
+
+## Download files from the Hub
+
+Repositories store versioned model and dataset files anyone can download. You can use the [`hf_hub_url`] function to download files. This function will download and cache a file on your local disk. The next time you need that file, it will load from your cache, so you don't need to re-download it.
```

Comment:
```suggestion
Repositories on the hub are git version controlled and users can download a single file, or the whole repository. You can use the [`hf_hub_download`] function to download files. This function will download and cache a file on your local disk. The next time you need that file, it will load from your cache, so you don't need to re-download it.
```